src/main/java/uk/gov/ons/ctp/response/action/domain/repository/ActionRepository.java

```diff
@@ -102,5 +102,5 @@
    * @param partyId UUID of associated party
    * @return Action returns an action is
    */
-  Action findOneByCaseIdAndActionRuleFKAndPartyId(UUID caseId, Integer actionRuleFK, UUID partyId);
+  Boolean existsByCaseIdAndActionRuleFKAndPartyId(UUID caseId, Integer actionRuleFK, UUID partyId);
```

Comment:
you can use the primitive boolean, and you should prefer it because it only has two possible values (true and false) instead of three (Boolean.TRUE, Boolean.FALSE and null)